libbeat/outputs/logstash/client.go

```diff
@@ -71,6 +72,16 @@ func newLumberjackClient(
 	}
 }
 
+func (l *lumberjackClient) Connect(timeout time.Duration) error {
```

Comment:
Trying to follow how this change is related to the other changes? Is this Method used? Same for close.